src/components/central-program.js

```diff
@@ -41,73 +53,77 @@ const CentralProgram= ({ data }) => {
                 </RequireWideScreen>
             }
             */}
-            <Container>
-                <Row>
-                    <Col md={9} xl={6} className="mx-auto">
-                        <div className="pt-4 pt-sm-4">
-                            <h2 className="pb-3 pt-sm-3 pt-md-2">{`Staff Roles (${data.site.siteMetadata.latestSchoolYear})`}</h2>
-                            <StaffRolesTable data={centralProgram.staff_roles} />
-                        </div>
-                        <div className="pt-4">
-                            <h2 className="pb-3">{`Staff Labor Unions (${data.site.siteMetadata.latestSchoolYear})`}</h2>
-                            <StaffLaborUnionsChart data={centralProgram.staff_bargaining_units}/>
-                            <StaffLaborUnionsTable data={centralProgram.staff_bargaining_units} />
-                        </div>
-                    </Col>
-                </Row>
-            </Container>
-        </div>
-        </Layout>
-    )
+        <Container>
+          <Row>
+            <Col md={9} xl={6} className="mx-auto">
+              <div id={`${ELEMENT_NAME_PREFIX}-1`} className="pt-4 pt-sm-4">
+                <h2 className="pb-3 pt-sm-3 pt-md-2">{`Staff Roles (${data.site.siteMetadata.latestSchoolYear})`}</h2>
+                <StaffRolesTable data={centralProgram.staff_roles} />
+              </div>
+              <div id={`${ELEMENT_NAME_PREFIX}-2`} className="pt-4">
+                <h2 className="pb-3">{`Staff Labor Unions (${data.site.siteMetadata.latestSchoolYear})`}</h2>
+                <StaffLaborUnionsChart
+                  data={centralProgram.staff_bargaining_units}
+                />
+                <StaffLaborUnionsTable
+                  data={centralProgram.staff_bargaining_units}
+                />
+              </div>
+            </Col>
+          </Row>
+        </Container>
+      </div>
+    </Layout>
+  )
 }
 
 export const query = graphql`
   query($code: Int!) {
     site {
-        siteMetadata {
-          latestSchoolYear
-        }
+      siteMetadata {
+        latestSchoolYear
+      }
     }
-    centralProgramsJson(code: {eq: $code }) {
-        name
-        budget
-        remaining_budget_percent
-        eoy_total_fte
-        eoy_total_positions
-        spending
-        year
-        code
-        staff_roles {
-            eoy_total_positions_for_role
-            role_description
-        }
-        staff_bargaining_units {
-            eoy_total_positions_for_bu
-            description
-            abbreviation
-        }
+    centralProgramsJson(code: { eq: $code }) {
+      name
+      budget
+      remaining_budget_percent
+      eoy_total_fte
+      eoy_total_positions
+      spending
+      year
+      code
+      staff_roles {
+        eoy_total_positions_for_role
+        role_description
+      }
+      staff_bargaining_units {
+        eoy_total_positions_for_bu
+        description
+        abbreviation
+      }
```

Comment:
Sorry, the code formatting is my fault. I've had my tab width set to 4. Didn't realize we had a Prettier config. It must have come with the Gatsby starter project. I'll try to use that from now on.

Reply:
No problem!